rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/get_scan/action.py

```diff
@@ -20,11 +20,11 @@ def run(self, params={}):
 
         try:
             response = self.connection.ivm_cloud_api.call_api("scan/" + scan_id, "GET")
-            return response
+            return response[1]
         except requests.RequestException as e:
             self.logger.error(e)
             raise PluginException(
                 cause=f"Failed to get a valid response from InsightVM with given asset ID '{scan_id}'.",
-                assistance=f"Response was {response.request.body}",
-                data=response.status_code,
+                assistance=f"Response was {response[1].request.body}",
```

Comment:
Does this line work? In this line (https://github.com/rapid7/insightconnect-plugins/pull/881/files/74d838a13c4e50bd843e3a2e7963fd91610f0cf0..77bd58beb3fdefe2c051f0ec08c0e9ca1f24eeef#diff-9bd5f544b673fabcaedb1df4c2932ffea8f84f763f699ebf10c0de8b547ba4efR44) you are returning a dictionary which is `response[1]` here - I don't think this has a `request` property on it.